moveit_experimental/moveit_jog_arm/package.xml

```diff
@@ -7,6 +7,7 @@
 
   <maintainer email="blakeanderson@utexas.edu">Blake Anderson</maintainer>
   <maintainer email="andyz@utexas.edu">Andy Zelenak</maintainer>
+  <maintainer email="adam.pettinger@utexas.edu">Adam Pettinger</maintainer>
```

Comment:
@AdamPettinger, do you really want to become a maintainer of this package or do you consider yourself as a contributor?

Reply:
I certainly do not need to be